### PR TITLE
DOC: Cleanup of Sequences module text

### PR DESCRIPTION
**Pull request by jamesobutler**
This is a followup to recent commit f3899b267c438c20023c887e8431f53236be0ca3. This provides a slight cleanup of the text in this module documentation to improve readability.

### CODE REVIEW

File: Docs/user_guide/modules/sequences.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ Authors:
 
 ## Acknowledgements
 
-This work is funded in part by An Applied Cancer Research Unit of Cancer Care Ontario with funds provided by the Ministry of Health and Long-Term Care and the Ontario Consortium for Adaptive Interventions in Radiation Oncology (OCAIRO) to provide free, open-source toolset for radiotherapy and related image-guided interventions.
+This work is funded in part by an Applied Cancer Research Unit of Cancer Care Ontario with funds provided by the Ministry of Health and Long-Term Care and the Ontario Consortium for Adaptive Interventions in Radiation Oncology (OCAIRO) to provide a free, open-source toolset for radiotherapy and related image-guided interventions.
```
Comment (jcfr) [inline Docs/user_guide/modules/sequences.md:94]:
```suggestion
This work is funded in part by the Applied Cancer Research Unit of Cancer Care Ontario with funds provided by the Ministry of Health and Long-Term Care and the Ontario Consortium for Adaptive Interventions in Radiation Oncology (OCAIRO) to provide a free, open-source toolset for radiotherapy and related image-guided interventions.
```

Comment (jamesobutler) [reply Docs/user_guide/modules/sequences.md:94]: I believe “an Applied Cancer Research Unit of Cancer Care Ontario“ is correct over “the Applied Cancer Research Unit of Cancer Care Ontario“ as there is more than one research unit of Cancer Care Ontario and the specific one is not being mentioned here.

I Google searched and found the usage of “On-PROST is an Applied Cancer Research Unit of Cancer Care Ontario, with funds provided by the Ministry of Health and Long-Term Care.”

Comment (jcfr) [reply Docs/user_guide/modules/sequences.md:94]: Thanks for looking this up. Let's ignore my suggestion then.

**Comment by lassoan** [comment]
Thank you, I've merged these suggestions along with other formatting and configuration fixes in https://github.com/Slicer/Slicer/pull/5697

**Pull request closed without merging**